### DISCUSSION TITLE
node-collector marked for deletion

### DISCUSSION THREAD

**Discussion by gaeljw** [aquasecurity/trivy-operator#1222] [Help & Support]
Hi folks,

We are using kwatch to be notified of some error events from our pods and we keep getting the following errors with node-collector pods (at least once per hour):

```
Name:
node-collector-ff974d987-vcdrm

Container:
node-collector

Namespace:
trivy-system

Reason:
ContainerStatusUnknown

Events:
[2023-05-15 19:58:58 +0000 UTC] TaintManagerEviction Marking for deletion Pod trivy-system/node-collector-ff974d987-vcdrm
[2023-05-15 19:58:58 +0000 UTC] TaintToleration Predicate TaintToleration failed
```

To be honest, I'm completely new to this and I'm not sure if this indicates something wrong with Trivy, Trivy Operator or it's just somehow expected and it's kwatch that raise false alarms.

Any help appreciated :)

Thanks!

**Comment by chen-keinan** [comment]
@gaeljw have you tried to set up [scanJobTolerations](https://github.com/aquasecurity/trivy-operator/blob/main/deploy/helm/values.yaml#L148) ?

**Reply by gaeljw** [reply]
Not yet, will look into it, thanks :)
